modules/gltf/gltf_document.cpp

```diff
@@ -3096,7 +3096,7 @@ Error GLTFDocument::_parse_images(Ref<GLTFState> p_state, const String &p_base_p
 		if (d.has("uri")) {
 			// Handles the first two bullet points from the spec (embedded data, or external file).
 			String uri = d["uri"];
-			image_name = uri;
+			image_name = itos(i);
```

Comment:
Might need to keep image_name = uri here.

and 

move image_name = itos(i) under the data section.

Doing testing.